registry_resolver/build.rs

```diff
@@ -1,14 +1,20 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let package_path = "buf.build/knox-networks/registry-mgmt";
-    let commit = "f7ff6f57030c418e886459a18b35645e";
-    let std::process::Output { status, stderr, .. } = std::process::Command::new("buf")
-        .arg("generate")
-        .arg(format!("{package_path}:{commit}"))
-        .output()?;
-
-    if !status.success() {
-        return Err(String::from_utf8_lossy(&stderr).into());
+    #[cfg(not(feature = "ci_ffi_build"))]
```

Comment:
Why is this disabled during the ffi build?

Reply:
For some strange reason cross is not able to build the project with buf being built in the build.rs. I've tried to figure out why, most likely it lacks some envvars, but eventually workaround with building buf separately, before the FFI build worked out.

Reply:
That's strange, well good to know!